src/coverage.ts

```diff
@@ -0,0 +1,81 @@
+import { once } from 'events';
+import { fork, execSync } from 'child_process';
+import path from 'path';
+import { existsSync, renameSync } from 'fs';
+import { removeSync, moveSync, copySync } from 'fs-extra';
+import exitHook from 'exit-hook';
+
+export function cleanUp(): void {
+  if (existsSync('./contracts-backup/')) {
+    moveSync('./contracts-backup', './contracts', { overwrite: true });
+  }
+  removeSync('./contracts-backup/');
+  removeSync('./build/contracts/');
+  removeSync('./.coverage_artifacts/');
+  removeSync('./.coverage_contracts/');
```

Comment:
Instead of hardcoding these paths we should use what was returned by `utils.getTempLocations`.

That function has a `config` argument so we should set up `exitHook(cleanUp)` inside the `runCoverage` function, once the `config` object has been built. I actually think it makes more sense to set up the hook when the function runs rather than at the top level.